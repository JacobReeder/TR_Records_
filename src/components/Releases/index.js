import React from 'react';


const Releases = () => {

    return (
        <div>
            <hi>
                <div></div>
            </hi>

            <h3></h3>
        </div>
    )
}

export default Releases;