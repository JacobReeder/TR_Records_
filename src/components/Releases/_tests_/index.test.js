// __tests__/Releases.test.js
/*import React from 'react'
import { render, cleanup } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import Releases from '..'
const portrait = { name: "portraits", description: "Portraits of people in my life" };

afterEach(cleanup)

describe('Releases component', () => {

  it('renders', () => {
    render(<Releases currentCategory={portrait} />);
  });

  it('matches snapshot', () => {
    const { asFragment } = render(<Releases currentCategory={portrait} />)
    expect(asFragment()).toMatchSnapshot()
  })
});

it('renders', () => {
  const { getByTestId } = render(<Releases currentCategory={portrait} />)
  expect(getByTestId('h1tag')).toHaveTextContent('Portraits')
})*/