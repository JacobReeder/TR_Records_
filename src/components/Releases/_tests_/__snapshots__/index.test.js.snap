// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Releases component matches snapshot 1`] = `
<DocumentFragment>
  <section>
    <h1
      data-testid="h1tag"
      id="releases"
    >
      Portraits
    </h1>
    <div>
      <p>
        Portraits of people in my life
      </p>
      <div
        class="flex-row"
      />
    </div>
  </section>
</DocumentFragment>
`;
